spec.html

```diff
@@ -4764,6 +4764,34 @@ <h1>GetFunctionRealm ( _obj_ )</h1>
         <p>Step 5 will only be reached if _obj_ is a non-standard function exotic object that does not have a [[Realm]] internal slot.</p>
       </emu-note>
     </emu-clause>
+
+    <emu-clause id="sec-copydataproperties" aoid="CopyDataProperties">
+      <h1>CopyDataProperties (_target_, _source_, _excluded_)</h1>
+      <p>When the abstract operation CopyDataProperties is called with arguments _target_, _source_ and _excluded_, the following steps are taken:</p>
+      <emu-alg>
+        1. Assert: Type(_target_) is Object.
+        1. Assert: Type(_excluded_) is List.
+        1. If _source_ is *undefined* or *null*, let _keys_ be a new empty List.
+        1. Else,
+          1. Let _from_ be ! ToObject(_source_).
+          1. Let _keys_ be ? _from_.[[OwnPropertyKeys]]().
+        1. Repeat for each element _nextKey_ of _keys_ in List order,
```

Comment:
The wording of spec language has been getting regularized and changed a little since this proposal was written. The modern wording is, `For each element _nextKey_ of _keys_, do`. For an example, see [the current definition of Object.assign](https://tc39.github.io/ecma262/#sec-object.assign).

Reply:
It's also okay to say `For each element _nextKey_ of _keys_ in List order, do`.